report_verification.md

```diff
@@ -525,10 +525,9 @@ significant design and exploration.
 ## Shared Storage in Fenced Frames
 
 When a shared storage operation is run from a fenced frame instead of a
-document, we can’t simply set a contextual ID. Winning ads of FLEDGE auctions
-are required to be _k_-anonymous and can’t communicate with their embedder. So,
-any high entropy contextual ID could not be joined to information outside the
-Fenced Frame.
+document, we can no longer set a contextual ID. Arbitrary information can be put
```

Comment:
ditto: maybe something like "Any cross-site information the fenced frame has could be embedded in the context ID, so the ability..."

Reply:
Rephrased this section.